packages/paste-website/src/pages/roadmap/index.mdx

```diff
@@ -78,21 +78,21 @@ After this is released, if you see visual or functional issues in the doc site,
   <CalloutText>Codenamed <strong>that one after Oh! Point Three!</strong> This release is expected to land in the <em>third week of November</em>. This release is <em>not locked in</em>, but here are candidates for the release.</CalloutText>
 </Callout>
 
-- Polish and Document Truncate Component
-- Polish and Document Media object Component
+- Polish and document truncate component
+- Polish and document media object component
```

Comment:
Media Object -> capitalized

Dont want to comment on each line-item, but the pattern should repeat itself here :)

Reply:
@TheSisb I've gone through and capitalized the component name.